lib/const.rb

```diff
@@ -452,7 +454,9 @@
     { pkg_name: 'pthread_stubs', anitya_pkg: 'libpthread-stubs', comments: '' },
     { pkg_name: 'py3_atspi', anitya_pkg: 'pyatspi', comments: '' },
     { pkg_name: 'python3', anitya_pkg: 'python313', comments: '' },
+    { pkg_name: 'qt5_base', anitya_pkg: 'qt', comments: '' },
```

Comment:
Is there a qt5 specific mapping? 

Also we use the kids backports branch for qt5...

Reply:
I don't believe we have a qt5 package.  Just qt5_* packages.
```
$ ls qt5*
qt5_base.rb    qt5_declarative.rb   qt5_location.rb    qt5_networkauth.rb    qt5_quickcontrols2.rb  qt5_serialport.rb  qt5_tools.rb    qt5_webchannel.rb  qt5_webglplugin.rb  qt5_x11extras.rb
qt5_charts.rb  qt5_imageformats.rb  qt5_multimedia.rb  qt5_quickcontrols.rb  qt5_script.rb          qt5_svg.rb         qt5_wayland.rb  qt5_webengine.rb   qt5_websockets.rb   qt5_xmlpatterns.rb
```

Reply:
Or do you mean is there an Anitya package with just version 5?  I don't think so.